### PR TITLE
[plugin] add support for usePaginatedElementData

### PR DESCRIPTION
**Pull request by xyin96**

### CODE REVIEW

File: src/react/hooks.ts [hunk 1]
```diff
@@ -98,6 +98,30 @@ export function useElementData(id: string): WorkbookElementData {
   return data;
 }
 
+/**
+ * Provides the latest data values from corresponding sheet with a callback to fetch more
```
Comment (Pearce-Ropion) [inline src/react/hooks.ts:102]: Add how many are fetched per page to the comment

File: src/react/hooks.ts [hunk 2]
```diff
@@ -98,6 +98,30 @@ export function useElementData(id: string): WorkbookElementData {
...
+    }
+  }, [id]);
+
+  useEffect(() => {
+    if (id) {
+      return client.elements.subscribeToElementData(id, setData);
+    }
+  }, [client, id]);
+
+  return useMemo(() => [data, loadMore], [data]);
```
Comment (Pearce-Ropion) [inline src/react/hooks.ts:122]: The expectation is that the user should immediately destructure the result of this hook, so it doesn't need to be memo-ed

```suggestion
  return [data, loadMore];
```

File: src/react/hooks.ts [hunk 3]
```diff
@@ -98,6 +98,30 @@ export function useElementData(id: string): WorkbookElementData {
   return data;
 }
 
+/**
+ * Provides the latest data values from corresponding sheet with a callback to fetch more
+ * @param {string} id Sheet ID to get element data from
+ * @returns {WorkbookElementData} Element Data for corresponding sheet, if any
+ */
+export function usePaginatedElementData(id: string): WorkbookElementData {
```
Comment (Pearce-Ropion) [inline src/react/hooks.ts:106]:
```suggestion
export function usePaginatedElementData(id: string): [WorkbookElementData, () => void] {
```

**Review by Pearce-Ropion — COMMENTED** [review]
Can you update the README?

File: README.md [hunk 4]
```diff
@@ -641,7 +663,7 @@ Returns a given variable's value and a setter to update that variable
 function useVariable(
   variableId: string,
 ): [WorkbookVariable | undefined, (...values: unknown[]) => void];
-```
+````
```
Comment (Pearce-Ropion) [inline README.md:666]: Intentional?

File: src/client/initialize.ts [hunk 5]
```diff
@@ -190,6 +190,9 @@ export function initialize<T = {}>(): PluginInstance<T> {
           void execPromise('wb:plugin:element:unsubscribe:data', id);
         };
       },
+      fetchMoreElementData(id) {
```
Comment (Pearce-Ropion) [inline src/client/initialize.ts:193]: Type this

**Review by Pearce-Ropion — DISMISSED** [review]

File: src/client/initialize.ts (hunk 5 above)
Comment (peternandersson) [reply src/client/initialize.ts:193]: All of these functions are untyped at the moment. I think I'm going to go through in a follow-up PR and add typing, rename these to `configId` and add a runtime check for `undefined` on these for better error messaging.

**Review by Pearce-Ropion — APPROVED** [review]

**Pull request merged**